examples/PythonProvider/python-django-webapp-sample/quickstartproject/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for quickstartproject project.
+
+Generated by 'django-admin startproject' using Django 4.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+import os
+from pathlib import Path
+from azure.appconfiguration.provider import load_provider, AzureAppConfigurationKeyVaultOptions
+from azure.identity import DefaultAzureCredential
+
+ENDPOINT = "your-endpoint"
```

Comment:
These 4 lines are the only real App Configuration thing, I think you should add a comment to highlight it. And make a note of it in the project readme.